packages/react/src/components/Table/TableHead/FilterHeaderRow/_filter-header-row.scss

```diff
@@ -13,22 +15,12 @@
   }
 }
 
-.#{$iot-prefix}--filter-header-row--header-width {
```

Comment:
This class, that probably at some point should have been active if there was a table-header-width, was actually active if there was NO table-header-width. And if even if it would have been the other way around, the `var('--table-header-width')` stuff doesn't work since the variables are within singe quotes. I tested all the affected stories and couldn't find any changes breaking when removing this.